src/app/shared/post-capture-card/post-capture-card.component.html

```diff
@@ -6,8 +6,8 @@
       (transaction$ | async)?.fulfilled_at | date: 'short'
     }}</mat-card-subtitle>
     <span class="toolbar-spacer"></span>
-    <button [hidden]="!sharable" (click)="share()" mat-icon-button>
-      <mat-icon>share</mat-icon>
+    <button [hidden]="!sharable" (click)="openOptionsMenu()" mat-icon-button>
+      <mat-icon>toc</mat-icon>
```

Comment:
I believe `more_vert` icon should be more appropriate. This is not a mandatory change request however.